cranelift/codegen/meta/src/gen_settings.rs

```diff
@@ -70,6 +71,27 @@ fn gen_constructor(group: &SettingGroup, parent: ParentGroup, fmt: &mut Formatte
     fmtln!(fmt, "}");
 }
 
+/// Generate an implementation of Default for ease-of-use.
+fn gen_default(group: &SettingGroup, parent: ParentGroup, fmt: &mut Formatter) {
```

Comment:
Same comment as below; could this be implemented in Rust with the same effect but making it easier to maintain?